plugins/pi_ir_barrier.py

```diff
@@ -0,0 +1,41 @@
+from .io_base import IOBase
+import RPi.GPIO as GPIO
+import time
+import logging
+
+from foos.config import ir_pin_a
```

Comment:
just for general knowledge, this should be: `from foos.config import ir_pin_a, ir_pin_b, ir_dedup_seconds`